docs/items/globallootmodifiers.md

```diff
@@ -0,0 +1,129 @@
+Global Loot Modifiers
+===========
+
+Global Loot Modifiers are a data-driven method of handling modification of harvested drops without the need to overwrite dozens to hundreds of vanilla loot tables or to handle effects that would require interactions with another mod's loot tables without knowing what mods may be loaded. Global Loot Modifiers are also stacking, rather than last-load-wins as modifications to loot tables would be.
+
+Registering a Global Loot Modifier
+-------------------------------
+
+You will need 3 things:
+1. Create a global_loot_modifiers.json file at /data/forge/loot_modifiers/
+	This will tell Forge about your modifiers and works similar to [tags].
```

Comment:
I don't think the link to tags will work like this. Don't you need to type it as `[tags][tags]`?

Reply:
See: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links
The type I'm using is `Or leave it empty and use the [link text itself].`
The URL gets defined a bit further down.